src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -950,16 +957,14 @@ public synchronized void start() {
         }
 
         if (errorCount == 0) {
-
+          
             encoder.setContext(getContext());
             if (!encoder.isStarted()) {
                 encoder.start();
             }
 
-            /*
-             * Increase the core size to handle the reader thread
-             */
-            int threadPoolCoreSize = getThreadPoolCoreSize() + 1;
+            int threadPoolCoreSize = 1;
+
             /*
```

Comment:
This is an extreme nit, but please add the comment back about the reader thread . This will help future us understand why it is initialized 1.

Reply:
You are right... I see what you mean.